.github/workflows/merge-commits.yml

```
@@ -72,13 +72,37 @@ jobs:
           git fetch --all
           git reset --hard ${{needs.dequeue.outputs.SHA}}
           git push --set-upstream origin main-next-${{needs.dequeue.outputs.SHA}}
+  merge-next:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      packages: write
+      actions: write
+      checks: write
+      issues: write
+      pull-requests: write
+      statuses: write
+    needs: [ create-branch ]
+    steps:
+      - name: Git checkout
+        uses: actions/checkout@7884fcad6b5d53d10323aee724dc68d8b9096a2e # pin@v2
+        with:
+          fetch-depth: '0'
+      - name: Merge next
+        run: |
+          git config user.name $USERNAME
+          git config user.email $EMAIL
+          git fetch origin
+          git checkout main-next-${{needs.dequeue.outputs.SHA}}
+          git merge $TARGET_BRANCH -X --no-ff
```

You need `theirs`, no?

```suggestion
          git merge $TARGET_BRANCH -X theirs --no-ff
```

---

I think you can remove the no-ff too. If a ff works, then great! It shouldn't ever work, but if it does then the merge is even more trivial :)